docs/developing/managing-focus.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Managing Focus
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <title>Managing Focus</title>
+  <meta
+    name="description"
+    content="Learn how to manage focus in Ionic applications using the setFocus API instead of the autofocus attribute."
+  />
+</head>
+
+## Autofocus Attribute
+
+The `autofocus` attribute is a standard HTML attribute that allows developers to set focus to an element when a page loads. This attribute is commonly used to set focus to the first input element on a page. However, the `autofocus` attribute can cause issues in routing applications when navigating between pages. This is because the `autofocus` attribute will set focus to the element when the page loads, but will not set focus to the element when the page is revisited.
+
+## Managing Focus
```

Comment:
Having this header be the same as the page title looks awkward. It also feels strange to offer `autofocus` and `setFocus` as two top-level sections; it looks like we're presenting the two as equal options with their own benefits and tradeoffs, when really we're explaining why `autofocus` doesn't work for most Ionic apps and showing how to use `setFocus` instead.

If it were me, I would do the following:
1. Remove the "Autofocus Attribute" header and go straight into the introductory content instead.
2. Trim the `autofocus` description paragraph to focus more on why it shouldn't be used. Link to the MDN docs for the attribute instead so developers can use that to learn more if they don't already know what it is.
3. Remove the "Managing Focus" header as well.
4. Raise all remaining headers to `##` instead of `###`.

An alternative would be to rename the `##` headers to something like "Downsides of `autofocus`" and "The `setFocus` API" respectively. Names could use workshopping, of course, but something that better shows the intent of the page. Personally though, I would rather draw the focus (hehe) of the page's outline entirely onto `setFocus` so things feel less cluttered.